content/en/security-audit/audit-2025-1.md

```diff
@@ -0,0 +1,7 @@
+---
+image: images/ada-security-audit.png
```

Comment:
It seems not showing under the year 2025, instead, it is under the 2023 with previous audits

Reply:
@yizha1 Fixed. Preview link: https://deploy-preview-439--notarydev.netlify.app/security-audit/. 

Only one small issue is that the 2025 audit report is not aligned with others https://deploy-preview-439--notarydev.netlify.app/security-audit/. This will take a longer time to improve the front-end code. It does not impact anything so we can merge it as the first iteration.